app/views/partials/channels-tab-url-rewriting.html

```diff
@@ -0,0 +1,115 @@
+      <label><span style="color: #a94442;">*</span> To Port</label><br />
+      <input ng-model="newUrlRewrite.toPort" class="form-control" placeholder="To Port" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{ngErrorUrlRewrite.portError}}</div>
+    </div>
+  </div>
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;">
+      <label>
+        Path Transform
+        <i ng-show="uiSettings.showTooltips" tooltip-html-unsafe="Transform the path by using the following substitute expression format:<br /> s/from/to/g" class="glyphicon glyphicon-question-sign"></i>
+      </label><br />
+      <input ng-model="newUrlRewrite.pathTransform" class="form-control" placeholder="s/from/to/g " type="text" />
+    </div>
+  </div>
+  
+
+  <div style="margin-top: 10px;">
+    <button type="button" class="btn btn-default" ng-click="cancelUrlRewriteAddEdit()">Cancel</button>
+    <button type="button" class="btn btn-primary" ng-click="saveUrlRewrite( oldUrlRewriteIndex )" style="margin-left: 5px;">Set Rewrite Rule</button>
+
+    <!-- hasErrors Alerts -->
+    <alert style="display: inline-block; margin: 0px 0px 0px 5px; padding: 8px;" ng-repeat="alert in alerts.hasErrorsUrlRewrite" type="alert.type">
+      <i class="glyphicon glyphicon-warning-sign"></i>{{alert.msg}}
+    </alert>
+    <!-- hasErrors Alerts -->
+  </div>
+</div>
+
+
+<div ng-if="!urlRewriteAddEdit">
+
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;">
+      <label>
+        URL Rewriting enabled?
+        <i ng-show="uiSettings.showTooltips" tooltip="URL rewriting allows the OpenHIM to look for URLs in a response and rewrite them so that they point to the correct location. These rules can be specified explicitly below." class="glyphicon glyphicon-question-sign"></i>
+      </label><br />
+      <label class="btn btn-primary" ng-model="channel.rewriteUrls" btn-radio="true">True</label>
+      <label class="btn btn-primary" ng-model="channel.rewriteUrls" btn-radio="false">False</label>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;">
+      <label>
+        Add Auto Rewrite Rules?
+        <i ng-show="uiSettings.showTooltips" tooltip="Determines whether automatic rewrite rules are used. These rules enabled URLs to be automatically rewritten for any URLs that points to a host that the OpenHIM proxies (any host on a primary route). These can be overridden by user specified rules if need be." class="glyphicon glyphicon-question-sign"></i>
+      </label><br />
+      <label class="btn btn-primary" ng-model="channel.addAutoRewriteRules" btn-radio="true">True</label>
+      <label class="btn btn-primary" ng-model="channel.addAutoRewriteRules" btn-radio="false">False</label>
+    </div>
+  </div>
+
+  <alert ng-if="channel.rewriteUrlsConfig.length == 0" type="'warning'">No URL Rewrites have been added</alert>
```

Comment:
If this use the term "URL Rewrite Rule" instead of "URL Rewrites", the same goes for where ever else this is used. I think missed a few places on the first round of CR.